### DISCUSSION TITLE
How did you handle the focus?

### DISCUSSION THREAD

**Discussion by todde15** [mcNets/UnoKeyboard#7] [Q&A]
Hi I am trying to build a similar on screen keyboard, but I really can't figure out how to keep the focus where I want to write, the focus automatically switches to the gui, how did you do it?

**Comment by mcNets** [comment]
Using FocusManager events.

Have a look at HandleFocusManager property:

https://github.com/mcNets/UnoKeyboard/blob/3f64795fc2547dace5fa8f6173734987cf44858a/src/UnoKeyboard/Controls/KeyboardControl.Properties.cs#L286-L300

**Comment by todde15** [comment]
Thank you so much for the answer
